x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/custom_assets_extension.tsx

```diff
@@ -42,7 +62,23 @@ export const CspCustomAssetsExtension = () => {
     },
   ];
 
-  return <CustomAssetsAccordion views={views} initialIsOpen />;
+  return (
+    <>
+      {search.length === 0 ? (
```

Comment:
Following my comment about changing useLocation to useParams we can update the logic, (I also suggest removing the ternary to make it more readable)


```js
if (!integration) {
	return (
				<>
						<CustomAssetsAccordion views={views} initialIsOpen title="Cloud Posture" />
						<EuiSpacer size="m" />
          	<CustomAssetsAccordion
            	views={viewsCNVM}
            	initialIsOpen
            	title="Cloud Native Vulnerability Management"
						/>
        </>
		)
}

if (integration === VULN_MGMT_POLICY_TEMPLATE) {
		return <CustomAssetsAccordion views={viewsCNVM} initialIsOpen />
} 

return <CustomAssetsAccordion views={views} initialIsOpen />;

```

*I didn't add the translation on the example, but that should be added.